Lib/_colorize.py

```diff
@@ -0,0 +1,45 @@
+import io
+import os
+import sys
+
+_COLORIZE = True
+
+
+class _ANSIColors:
+    BOLD_GREEN = "\x1b[1;32m"
+    BOLD_MAGENTA = "\x1b[1;35m"
+    BOLD_RED = "\x1b[1;31m"
+    GREEN = "\x1b[32m"
+    GREY = "\x1b[90m"
+    MAGENTA = "\x1b[35m"
+    RED = "\x1b[31m"
+    RESET = "\x1b[0m"
+    YELLOW = "\x1b[33m"
+
+
+def _can_colorize():
```

Comment:
Nit: since this is now a private module, I think I'd give all these public names. `from _colorize import _can_colorize` feels pretty ugly to me, and I think it's still obvious that it's private if it's called `_colorize.can_colorize`

```suggestion
COLORIZE = True


class ANSIColors:
    BOLD_GREEN = "\x1b[1;32m"
    BOLD_MAGENTA = "\x1b[1;35m"
    BOLD_RED = "\x1b[1;31m"
    GREEN = "\x1b[32m"
    GREY = "\x1b[90m"
    MAGENTA = "\x1b[35m"
    RED = "\x1b[31m"
    RESET = "\x1b[0m"
    YELLOW = "\x1b[33m"


def can_colorize():
```